pkg/controllers/clusterresourceplacement/controller_integration_test.go

```diff
@@ -344,6 +401,138 @@ var _ = Describe("Test ClusterResourcePlacement Controller", func() {
 				},
 			}
 			retrieveAndValidateClusterResourcePlacement(testCRPName, wantCRP)
+
+			By("Ensure placement complete metric was emitted with isCompleted False")
+			wantCompleteMetrics := []*prometheusclientmodel.Metric{
```

Comment:
This incomplete metric can be refactored into a function. There's code duplication.